test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -53,8 +53,13 @@ def ec2_pytorch_inference(image_uri, processor, ec2_connection, region):
     repo_name, image_tag = image_uri.split("/")[-1].split(":")
     container_name = f"{repo_name}-{image_tag}-ec2"
     model_name = "pytorch-densenet"
+    if processor is "eia":
+        image_framework, image_framework_version = get_framework_and_version_from_tag(image_uri)
+        if image_framework_version is "1.3.1":
```

Comment:
Please use `==` instead of `is` to compare a string variable to a string literal.
```suggestion
        if image_framework_version == "1.3.1":
```
Reasoning:
https://stackoverflow.com/questions/47778889/pylint-complains-about-comparing-a-string-to-a-literal-with-is
https://stackoverflow.com/questions/2209755/python-operation-vs-is-not